doc/Reference/Reactive/listfeed.md

```diff
@@ -46,7 +46,7 @@ For example:
 public IListFeed<string> Names => ListFeed.AsyncEnumerable(service.GetNames);
 ```
 
-Pull and push are explained more in the [feeds page](xref:Uno.Extensions.Mvux.Feeds#creation-of-feeds).
+Pull and push are explained more in the [feeds page](xref:Uno.Extensions.Reactive.Feed#sources:how-to-create-a-feed).
```

Comment:
I am not sure if this is the right way to use this link. It's supposed to go to a part of the page labeled:  "Sources: how to create a feed"

Reply:
fixed it

"when a heading contains a colon (:) in Markdown, you typically exclude the colon from the xref link (URL fragment). Similar to semicolons, most Markdown processors ignore colons and other punctuation when generating an ID for a heading."